scripts/backup

```diff
@@ -63,15 +63,24 @@ ynh_backup --src_path="/etc/php/$phpversion/fpm/pool.d/$app.conf"
 # Backup cron job
 ynh_backup --src_path="/etc/cron.d/$app"
 
+# Put Friendica in maintenance mode
+ynh_print_info --message="Enabling the maintenance mode for Friendica"
+/var/www/friendica/bin/console maintenance 1
```

Comment:
```suggestion
__INSTALL_DIR__/bin/console maintenance 1
```

Reply:
or better
```
(cd $install_dir && ynh_exec_as "$app" php$phpversion bin/console maintenance 1)
```

Reply:
Yes much better indeed! This works great I tested it so you can merge into the testing branch of you wish.